### PR TITLE
Cypress test for pagination

### PR DESCRIPTION
**Pull request by ppadti**
Closes: [RHOAIENG-5117](https://issues.redhat.com/browse/RHOAIENG-5117)

## Description
This PR aims to create a utility to test pagination. Follow up PR will have pagination for pipelines.

## How Has This Been Tested?
`npm run test`

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/__tests__/cypress/cypress/utils/pagination.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { tablePagination } from '~/__tests__/cypress/cypress/pages/components/Pagination';
+
+type PaginationProp = {
+  totalItems: number;
+  firstElement: string;
+  paginationVariant: string;
```
Comment (pnaik1) [inline frontend/src/__tests__/cypress/cypress/utils/pagination.ts:6]:
```suggestion
  paginationVariant:  'top' | 'bottom'

File: frontend/src/__tests__/cypress/cypress/utils/pagination.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  totalItems = 0,
+  firstElement = 'Test',
+  paginationVariant = 'top',
+}: PaginationProp): void => {
+  let pagination;
+  if (paginationVariant === 'top') {
+    pagination = tablePagination.top;
+  } else {
+    pagination = tablePagination.bottom;
+  }
```
Comment (pnaik1) [inline frontend/src/__tests__/cypress/cypress/utils/pagination.ts:19]:
```suggestion
 const pagination = paginationVariant === 'top' ? tablePagination.top : tablePagination.bottom;
```

File: frontend/src/__tests__/cypress/cypress/utils/pagination.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+type PaginationProp = {
+  totalItems: number;
+  firstElement: string;
+  paginationVariant: string;
+};
+
+export const testPagination = ({
+  totalItems = 0,
+  firstElement = 'Test',
+  paginationVariant = 'top',
```
Comment (pnaik1) [inline frontend/src/__tests__/cypress/cypress/utils/pagination.ts:12]: is the default value needed here?

**Review by pnaik1 — CHANGES_REQUESTED** [review]

File: frontend/src/__tests__/cypress/cypress/utils/pagination.ts (hunk 3 above)
Comment (ppadti) [reply frontend/src/__tests__/cypress/cypress/utils/pagination.ts:12]: I think, those are not required. I forgot to remove those. I will remove that.

**Pull request merged**